app/controllers/admin/enterprises_controller.rb

```diff
@@ -122,7 +122,8 @@ def for_order_cycle
       respond_to do |format|
         format.json do
           render json: @collection,
-                 each_serializer: Api::Admin::ForOrderCycle::EnterpriseSerializer, order_cycle: @order_cycle, spree_current_user: spree_current_user
+                 each_serializer: Api::Admin::ForOrderCycle::EnterpriseSerializer,
+                 order_cycle: @order_cycle, spree_current_user: spree_current_user
```

Comment:
I think this change is a bit confusing, I'd prefer something like this : 
```suggestion 
          render(
              json: @collection,
              each_serializer: Api::Admin::ForOrderCycle::EnterpriseSerializer,
              order_cycle: @order_cycle,
              spree_current_user: spree_current_user
          )
```